@types/ReactNative.ts

```diff
@@ -5,5 +5,6 @@ declare module "react-native" {
     export interface NativeModulesStatic {
         APIClient: NativeAPIClient;
         GenericClient: NativeGenericClient;
```

Comment:
Just noticed from the changes further below - these probably need to be updated to `RNNCAPIClient` etc if we're changing the names of the native modules.